### PR TITLE
graceful_exit on proxyerror

### PR DESCRIPTION
**Pull request by aviramha**

### CODE REVIEW

File: mirrord/layer/src/error.rs [hunk 1]
```diff
@@ -212,6 +212,11 @@ impl From<HookError> for i64 {
             HookError::SocketUnsuportedIpv6 => {
                 info!("{fail}")
             }
+            HookError::ProxyError(err) => {
+                graceful_exit!(
+                    "Proxy error, connectivity issue or a bug. Please report if not connectivity.\n{err}"
```
Comment (meowjesty) [inline mirrord/layer/src/error.rs:217]:
```suggestion
                    "Proxy error, connectivity issue or a bug. \n\
                    You may report it to us on https://github.com/metalbear-co/mirrord/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml \n{err}"
```

Comment (meowjesty) [reply mirrord/layer/src/error.rs:217]: We may want people to report this stuff to us? Or have you seen cases where it's just a small issue on the user side?

**Review by meowjesty — APPROVED** [review]
Should we always ask for a report?

**Comment by aviramha** [comment]
> Should we always ask for a report?

I think connection might be dropping which is okay and not a bug.

**Pull request merged**